### PR TITLE
Add koyanek's pin to the map

### PR DESCRIPTION
**Pull request by koyanek**
Adding QC to the map

### CODE REVIEW

**Comment by lokilontan** [comment]
Hello! It`s very easy, you will do it. Just do the file with all information from Course which the provide. Succes!

File: _pins/koyanek.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+githubhandle: koyanek
+latitude: 14.676041
+longhitude: 121.043700
```
Comment (crichID) [inline _pins/koyanek.json]: @koyanek oops, you have an extra letter in here. Just remove the h so it is spelled `longitude`.

Comment (crichID) [inline _pins/koyanek.json]: @koyanek you will need to add three dashes to the end of you file `---`.

**Comment by brianamarie** [comment]
@koyanek  :wave: Let us know if you want or need any help making these changes. If in a few days the changes still aren't made, one of the GitHub trainers may make the changes for you to keep the repository clean. ✨

**Comment by BenAkor** [comment]
@koyanek  _pins/koyanek.json seems duplicated so it is conflicting.

**Pull request closed without merging**